### DISCUSSION TITLE
DAI 2024-2025 Practical work 1 - Nathan Parisod - Maxime Lestiboudois

### DISCUSSION THREAD

**Discussion by Nat004499** [heig-vd-dai-course/heig-vd-dai-course#396] [Show and tell]
We would like to merge two files into one. One would be an image, the other one would be a sound file. The result will be an image with a music.

**Comment by ludelafo** [comment]
Seems good to me but we might come to you to ensure it is not too much/difficult to accomaplish with the available time. What do you think @hadrylouis?

**Comment by Nat004499** [comment]
Finally, we will be merging two or more text files. The following commands will be available:

 - Merge, split, or replace a text file.
 - Define the delimiter between the merged files (e.g., newline \n, space, hyphen -, etc.).
 - Display information about a text file (e.g., file size, weight, etc.).

**Comment by Nat004499** [comment]
We have finish our project, come see it there: https://github.com/Nat004499/Dai-Project-1.git
Here is the final commit we made: [ad67b40](https://github.com/Nat004499/Dai-Project-1/commit/ad67b40d7fe9df1a612b48a90b2103e726fe8dec)

**Reply by ludelafo** [reply]
Bonjour @Nat004499, @MaximeLesti,

Nous n'avons pas les accès à votre repo. Merci d'ajouter @hadrylouis, @GeraudSilvestri et moi-même pour que nous puissions y avoir accès pour le corriger.

**Reply by Nat004499** [reply]
Désolé pour ceci, ce devrait être réglé
